static/app/views/releases/list/index.tsx

```diff
@@ -343,9 +344,87 @@ class ReleasesList extends AsyncView<Props, State> {
     );
   }
 
+  renderAdoptionChart(activeDisplay: DisplayOption) {
```

Comment:
Should we change this to render only either null or ReleaseAdoptionChart and move the computations, releaseStatsRequest and all that inside ReleaseAdoptionChart?

Reply:
I think that's a good idea but I don't want to bloat this pr. Let me handle that in another one.